jpo-ode-core/src/main/resources/schemas/schema-bsm.json

```diff
@@ -299,64 +299,16 @@
                                     "required": [],
```

Comment:
Should this be on a different schema specific to redaction rather than this schema-bsm.json file? I believe this file is our full BSM which will continue to have all properties

Reply:
Yeah this needs to be its own schema if we want to keep this in the jpo-ode repository. We definitely want to maintain our old schema for BSM and have a different one for filtered BSM.

Reply:
I've created a new schema called 'schema-filtered-bsm.json' and linked it in the schemas README. The 'schema-bsm.json' file has been restored to its previous state.